test/jdk/java/lang/StrictMath/ExhaustingTests.java

```diff
@@ -66,7 +66,10 @@ public static void main(String... args) {
     private static long testUnaryMethods() {
         long failures = 0;
         UnaryTestCase[] testCases = {
-         // new UnaryTestCase("sqrt",  FdlibmTranslit::sqrt,  StrictMath::sqrt,  DEFAULT_SHIFT),
+            // Since sqrt is correctly rounded, could additionaly the
+            // transliteration or StrictMath sqrt methods to against
+            // Math::sqrt.
```

Comment:
I guess this comment has been messed up by some copy&paste activities?
Or is it a ChatGPT output?

Reply:
Fixed mangled text.